tools/eslint-rules/timer-arguments.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @fileoverview Require at least two arguments when calling setTimeout() or
+ * setInterval().
+ * @author Rich Trott
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function isTimer(name) {
+  return ['setTimeout', 'setInterval'].includes(name);
+}
+
+module.exports = function(context) {
+  return {
+    'CallExpression': function(node) {
+      const name = node.callee.name;
```

Comment:
I'm not sure if it matters, but `name` will be `undefined` for calls like `foo.bar()`.